pkg/sbom/v1/sbom_spdx_storage_format.go

```
@@ -22,39 +26,80 @@ const (
 	KubescapeOrganizationName = "Kubescape"
 	KubescapeNodeAgentName    = "KubescapeNodeAgent"
 	RelationshipContainType   = "CONTAINS"
+	directorySBOM             = "SBOM"
 )
 
+var spdxDataDirPath string
+
 type SBOMData struct {
-	spdxData                              spdxv1beta1.SBOMSPDXv2p3
+	spdxDataPath                          string
 	filteredSpdxData                      spdxv1beta1.SBOMSPDXv2p3Filtered
 	relevantRealtimeFilesBySPDXIdentifier sync.Map
 	newRelevantData                       bool
 	alreadyExistSBOM                      bool
+	instanceID                            instanceidhandler.IInstanceID
+}
+
+func init() {
+	wd, err := os.Getwd()
+	if err != nil {
+		logger.L().Ctx(context.GetBackgroundContext()).Error("failed to get working directory", helpers.Error(err))
```

I suggest this function should return an error in case of an error.